app/src/main/java/com/jerboa/ui/components/post/create/CreatePostScreen.kt

```diff
@@ -118,12 +119,24 @@ fun CreatePostScreen(
     }
 
     val (suggestedTitle, suggestedTitleLoading) =
-        when (val res = createPostViewModel.siteMetadataRes) {
-            ApiState.Empty -> MetaDataRes(null, false)
-            ApiState.Loading -> MetaDataRes(null, true)
-            is ApiState.Success ->
-                MetaDataRes(res.data.metadata.title, false)
-            else -> MetaDataRes(null, false)
+        // If its a torrent magnet link, use the dn param to extract the torrent title
+        if (uri.scheme == "magnet") {
```

Comment:
Can you add a constant for these strings.

Reply:
I did add a `ALLOWED_SCHEMES` list, but this one only needs to check one of them.